roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# rook
+
+container_registry_rook: quay.io
+rook_image: "{{ container_registry_rook }}/ceph/ceph"
+# renovate: datasource=docker depName=index.docker.io/rook/ceph
+rook_image_tag: "v17.2"
+
+## work directory inside the osism-ansible container:
+rook_work_directory: /tmp/rook/configuration
+## template directory containing rook CRD `*.yml` files, e.g. OSISM configuration directory
+rook_template_directory: "{{ role_path }}/templates"
+## kubernetes namespace in which the rook cluster should be installed
+## also see https://rook.io/docs/rook/v1.13/Storage-Configuration/Advanced/ceph-configuration/?h=#using-alternate-namespaces
+rook_namespace: rook-ceph
+rook_cluster_name: rook-ceph
+rook_mon_count: 3
+rook_mds_count: 2
+rook_mgr_count: 2
```

Comment:
Is it a best practice to only have 2 mgr services by default? At least in ceph-ansible and in the docs we always say use at least 3 mgr services.

Reply:
It really depends on the workload. I raised this to 3 to be backward compatible by default.